assertGiven.js

```diff
@@ -1,22 +1,27 @@
-module.exports = function assertGiven(gives, given){
-  if (!given) { 
-    throw new Error('create function should return a function or an object') 
+module.exports = function assertGiven (gives, given) {
```

Comment:
at the moment this only checks for shallow object matches, but it's possible `gives` and `given` are deeply nested. `libnested.each` could be helpful here (which short circuits on `return false`). also maybe include the missing path in the error. but anyways, nice to have. even this is really awesome.